voluncheer/chatroom/views.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.views import redirect_to_login
+from django.shortcuts import render
+
+from chatroom.models import Room
+
+
+# To be changed after UI is designed.
+def chat_homepage_view(request):
+    if not request.user.is_authenticated:
```

Comment:
There is a handy decorator that does this for you: https://docs.djangoproject.com/en/4.1/topics/auth/default/#the-login-required-decorator